weave/trace_server/calls_query_builder/calls_query_builder.py

```diff
@@ -1195,6 +1202,37 @@ def process_trace_id_filter_to_sql(
     return f" AND ({trace_cond} OR {trace_id_field_sql} IS NULL)"
 
 
+def process_thread_id_filter_to_sql(
+    hardcoded_filter: Optional[HardCodedFilter],
+    param_builder: ParamBuilder,
+    table_alias: str,
+) -> str:
+    """Pulls out the thread_id and returns a sql string if there are any thread_ids."""
+    if hardcoded_filter is None or not hardcoded_filter.filter.thread_ids:
```

Comment:
Can you add an explicit condition for `hardcoded_filter.filter.thread_ids`? Truthiness is dangerous in Python. Is it `len(...) > 0` or `is not None`?